demo/vaadin-router-getting-started-demos.html

```diff
@@ -85,6 +85,39 @@ <h3>Getting Started</h3>
       <code>this.location.pathname</code>).
     </p>
 
+    <h3>Using <code>this.location</code></h3>
+    <p>
+      If you use Polymer and JavaScript to build your view components,
+      the <code>location</code> property does not require a declaration.
+      It is recommended to add it to the <code>properties</code> of your
+      component for consistency, though:
+    </p>
+    <pre><code>static get properties() {
+  return {
+    // ...
+    location: Object
+  };
+}</code></pre>
+    <p>
+      For LitElement and TypeScript a declaration in the component is required.
+      Declare the <code>location</code> property in the class and initialize it
+      from the <code>router</code> Vaadin Router instance:
+    </p>
+    <pre><code>import {router} from './index';
+import {customElement, html, LitElement, property} from 'lit-element';
+
+@customElement('my-view')
+class MyViewElement extends LitElement {
+  @property({type: Object}) location = router.location;
```

Comment:
Why not something like:

```typescript
  @property({ type: Object })
  private location!: RouterLocation;
```

Reply:
Because, right now, I have no access to the `router`.

I'm using a init function like:
https://github.com/Polymer/pwa-starter-kit/compare/master...abdonrd:vaadin-router?expand=1#diff-15ee858ae5d5aa9b3e2846cfd1c4c466R3